Makefile

```diff
@@ -46,23 +49,27 @@ cleans          := $(addprefix clean-, $(platformtests))
 .PHONY: all
 all: ${builds}
 
+.PHONY: list-tests
+list-tests:
+	@echo "$(TESTS)"
+
```

Comment:
Why do you need that? Isn't target autocompletion enough to see the available tests?

Reply:
Fair point. But not everyone has auto-completion (capable of dealing with this `Makefile`) setup.

Reply:
Can you open a PR adding to the README the shell config that's needed for autocompletion to work on MacOS?

Reply:
Hmm, don't you want to set that up? 
```
zstyle ':completion::complete:make:*:targets' call-command true
autoload -U compinit && compinit
```

Maybe this can be done within nix (@potsrevennil?)

Reply:
Hm, this does not seem to work for me.

Reply:
Oh, it does, but extremely slow. Nevermind. I'll keep this in, but also add a note to the README.